styles/globals.css

```diff
@@ -36,26 +39,26 @@ input[type='number'] {
 	margin: 0;
 } */
 
-input[type='number'] {
-	-moz-appearance: textfield;
-	appearance: textfield;
+input[type="number"] {
+  -moz-appearance: textfield;
+  appearance: textfield;
 }
 
 input::-webkit-outer-spin-button,
 input::-webkit-inner-spin-button {
-	-webkit-appearance: none;
-	margin: 0;
+  -webkit-appearance: none;
+  margin: 0;
 }
 
 input:-webkit-autofill,
 input:-webkit-autofill:focus {
-	transition: background-color 600000s 0s, color 600000s 0s;
+  transition: background-color 600000s 0s, color 600000s 0s;
 }
 
 input[data-autocompleted] {
-	background-color: transparent !important;
+  background-color: transparent !important;
 }
 
 .balancedText {
-	text-wrap: balance;
+  text-wrap: balance;
```

Comment:
Why are we declaring classes in our global CSS when we can do it in tailwindcss? Any valid reasons as to why it must be done here?

Reply:
I think there are conflicts as it doesnt work that way, so I have no other way to maneuver

Reply:
You can add it via the tailwind config plugin system instead... We do this so that the global CSS file will only contain resets, and not be littered with custom classes that the tailwindcss intellisense can't recognize